Src/Web/Web.Shared.Net/ClientIpHeaderTelemetryInitializer.cs

```diff
@@ -109,7 +109,11 @@ private static bool IsCorrectIpAddress(string address)
             // Core SDK does not support setting Location.Ip to malformed ip address
             if (IPAddress.TryParse(address, out outParameter))
             {
-                // Also SDK supports only ipv4!
+                // Even thoug Core SDK supports IPv6 - we do not expect it here
+                // We already have logic to cut the port that simply cuts everything after ":"
+                // meaning it will not work for IPv6 when port was not appended to the IP address
+                // Need to support RFC https://tools.ietf.org/html/rfc7239 better and make sure that 
```

Comment:
nit: This comment block is difficult for me to parse. For a start, could you please add periods at the end of sentences?
"... it here."
"... IP address."
I don't understand "cut the port".
I think, after studying it for a while, you're saying that: IPv6 addresses have colons in them and we have existing code that (mistakenly) assumes that everything after a colon is a port number. For now, we'll ignore the problem, but eventually we'll need to implement support for RFC7239. This is tracked by issue number xxx.